backend/tracim_backend/lib/core/workspace.py

```diff
@@ -218,16 +218,40 @@ def get_one(self, workspace_id: int) -> Workspace:
                 "workspace {} does not exist or not visible for user".format(workspace_id)
             ) from exc
 
-    def get_one_by_label(self, label: str) -> Workspace:
+    def get_one_by_filemanager_filename(
+        self, filemanager_filename: str, parent: typing.Optional[Workspace] = None
+    ) -> Workspace:
         """
         get workspace according to label given, if multiple workspace have
         same label, return first one found.
```

Comment:
The docstring would gain to be updated, especially explaining what is a `filemanager_filename`.

Reply:
done.